webpack/assets/javascripts/react_app/components/HostsIndex/ActionKebab.scss

```diff
@@ -0,0 +1,4 @@
+.pf-c-menu.pf-m-flyout {
+  position: absolute;
+  top: 3.1em;
+}
```

Comment:
Please use more specific rule, for the host index, and add a space at the end

Reply:
actually might be better to "just" use a popper to do it properly https://v4-archive.patternfly.org/v4/demos/composable-menu/

Reply:
@jeremylenz what do you think of the popper?

Reply:
Just saw your comment. I have never in my life seen that part of the PF documentation. :see_no_evil:  I hate the idea of using a ref, but I can change it if you think that's better.

Reply:
I trust a popper just a bit more than `top: 3.1em;` :sweat_smile:

Reply:
That was easier than I thought.. updated